src/common/eth-providers/consensus-provider/consensus-provider.service.ts

```diff
@@ -48,7 +49,8 @@ export class ConsensusProviderService {
     beaconHeadFinalityCheckpoints: 'eth/v1/beacon/states/head/finality_checkpoints',
     blockInfo: (blockId: BlockId): string => `eth/v2/beacon/blocks/${blockId}`,
     beaconHeaders: (blockId: BlockId): string => `eth/v1/beacon/headers/${blockId}`,
-    balances: (stateId: StateId): string => `eth/v1/beacon/states/${stateId}/validators`,
+    validatorsState: (stateId: StateId): string => `eth/v1/beacon/states/${stateId}/validators`,
+    attestationCommittees: (stateId: StateId, epoch: Epoch): string => `eth/v1/beacon/states/${stateId}/committees?epoch=${epoch}`,
```

Comment:
`stateId` identifies the epoch, isn't it?

Reply:
Not always. Sometimes we want to get duties for epoch by another state. For example, `head` or particular state hash. These are two independent parameters

Reply:
As I can see, these parameters are dependent since the state has its point in time and the corresponding epoch.

And our use case is: given an epoch `N` and the closest state `A` we can choose some state `B`, where `B.latest_block > A.latest_block` and query epoch `N` data from the state `B` to check that the data for the epoch from the state `A` is canonical, right?

Reply:
Our case is that we get committees for epoch as of the last slot of this epoch. I prefer to leave current according to CL API specification

Reply:
I don't mean to keep it as is. But I didn't understand what you meant.

Reply:
These are two independent variables by specification, and they will remain so

Reply:
Discussed via voice com. TODO: change the high possible rewards calculation relative to the finalized state instead of the head.